README.md

```diff
@@ -306,6 +307,26 @@ In order to achieve a Zero Copy message delivery, applications need to both enab
     </profiles>
     ```
 
+### Large data transfer over lossy network
+
+In default, Fast DDS uses UDPv4 for the data communication over the network.
+UDP has its own merit for realtime capability, and many applications rely on UDP.
+But this depends on the application requirement, if it requires a more reliable network communication with large data samples over the lossy network, probably TCP flow control is more suitable.
+
+Fast DDS provides the configuration to modify the builtin transports via environmental variable `FASTDDS_BUILTIN_TRANSPORTS` as below.
+
+```bash
+export FASTDDS_BUILTIN_TRANSPORTS=LARGE_DATA
+```
+
+This adds TCP transport for data communication, and restricts UDP transport only for discovery process.
+This will improve the large data transmission over the lossy network environment.
```

Comment:
```suggestion
This `LARGE_DATA` mode adds a TCP transport for data communication, restricting the use of the UDP transport to the first part of the discovery process, thus achieving a reliable transmission with automatic discovery capabilities.
This will improve the transmission of large data samples over lossy networks.
```